drivers/net/ethernet/pensando/ionic/ionic_lif.c

```diff
@@ -275,9 +275,9 @@ static int ionic_qcq_enable(struct ionic_qcq *qcq)
 		ctx.cmd.q_control.index, ctx.cmd.q_control.type);
 
 	if (qcq->flags & IONIC_QCQ_F_INTR) {
+		napi_enable(&qcq->napi);
```

Comment:
> So the more I look at this I think the `Fixes:` is wrong on this change.
> 
> The `upstream-diff` mentioned is due to results from this change [e8797a0](https://github.com/ctrliq/kernel-src-tree/commit/e8797a058466b60fc5a3291b92430c93ba90eaff) `ionic: clean interrupt before enabling queue to avoid credit race`
> 
> So this CVE `Fixes:` should actually be addressing the above commit, ie this cve is a bugfix to a bugfix. I thiink it the right thing to do is attempt to bring in [e8797a0](https://github.com/ctrliq/kernel-src-tree/commit/e8797a058466b60fc5a3291b92430c93ba90eaff) as a preconditional to this CVE fix since this is trying to clean the queue before enabling it.

Acked

Reply:
Fixed in the next PR